packages/protocol/contracts/README.abis.md

```diff
@@ -0,0 +1,27 @@
+# Celo core contracts ABIS
+
+Smart contracts ABIS for the Celo protocol.
+
+## Usage
+
+### Installation
+
+```bash
+npm install @celo/abis@beta
+```
+
+or
+
+```bash
+yarn add @celo/abis@beta
+```
+
+### In your javascript or ts
+
+```js
+import Accounts from '@celo/abis/Accounts.json'
+```
+
+## License
+
+The contents of this package are licensed under the terms of the GNU Lesser Public License V3
```

Comment:
can we double check this?

Reply:
This is the same license that's in use by the contracts, I suppose the ABIs follow the same license. Happy to verify though.

Reply:
I have asked legal